next.config.js

```diff
@@ -1,34 +1,36 @@
 module.exports = (phase, { defaultConfig }) => {
   // Add raw loader for markdown files.
   const webpack = (config) => {
-    config.module.rules.push(
-      {
-        test: /\.md$/,
-        use: 'raw-loader'
-      }
-    )
+    config.module.rules.push({
+      test: /\.md$/,
+      use: 'raw-loader'
+    })
     return config
   }
   // If running in a development environment, API_BASE_URL will be undefined
   // so we use dotenv to load the env file. Otherwise, these are assigned in
   // now.json to secrets stored within the zeit.co environment.
   if (process.env.API_BASE_URL === undefined) {
-    require('dotenv').config({path: '.env.build'})
+    require('dotenv').config({ path: '.env.build' })
   }
-  // Assign all values from .env file here.
-  // NOTE: If any values are added to .env, they must be applied here.
+  // Assign all values from .env.build file here.
+  // NOTE: If any values are added to .env.build, they must be applied here.
   // TODO: Find a better way to handle this.
   const env = {
     API_BASE_URL: process.env.API_BASE_URL,
     API_DOCUMENTATION_URL: process.env.API_DOCUMENTATION_URL,
     API_KEY: process.env.API_KEY,
+    API_MANAGER_ENABLED: process.env.API_MANAGER_ENABLED,
     API_NAME: process.env.API_NAME,
     AUTH0_AUDIENCE: process.env.AUTH0_AUDIENCE,
     AUTH0_CLIENT_ID: process.env.AUTH0_CLIENT_ID,
     AUTH0_DOMAIN: process.env.AUTH0_DOMAIN,
-    OTP_UI_URL: process.env.OTP_UI_URL,
     OTP_FORUM_URL: process.env.OTP_FORUM_URL,
+    OTP_UI_URL: process.env.OTP_UI_URL,
+    PRIMARY_COLOR: process.env.PRIMARY_COLOR,
```

Comment:
I'm sure someone will try to use a bright color so we might eventually need a text color thing 😐.

Reply:
We can maybe use tiny color again to automatically generate a readable text color. For now I will add a note to the demo config

Reply:
Note added in 3a29c33ffe0b1276cc11e44d7ce6ffe766f99d71